lib/server/connect.js

```diff
@@ -155,17 +248,18 @@ Meteor.methods({
     // set unique id for each server sessions
     data.serverId = serverId;
 
-    // update timeline
-    // XXX: Try to send these to timeline which monitor the given access token
-    SubHandlers.timeline.forEach(function(sub) {
-      var id = 'id' + ++uniqueId;
-      sub.added('kdTimeline', id, {
-        browserId: browserId,
-        clientId: clientId,
-        data: data
-      });
-      sub.removed('kdTimeline', id);
-    });
+    var dataObj = {};
+    dataObj._id = 'serverTimeEvents';
+    dataObj[serverId] = {};
+    dataObj[serverId].browserId = browserId;
+    dataObj[serverId].clientId = clientId;
+    dataObj[serverId].data = JSON.stringify(data);
+
+    if(browserId && clientId) {
+      // update data to DB
+      KdData.remove({ _id: 'serverTimeEvents'});
+      KdData.insert(dataObj);
```

Comment:
I mean add this document to the KdData with a type like 'timeevents'. Then we can query them later on.